ui/app/src/Main.elm

```diff
@@ -53,8 +59,32 @@ init location =
 
                 _ ->
                     nullFilter
+
+        apiUrl =
+            flags
```

Comment:
If we get the url from the *Navigation* package, parsing flags should not be necessary anymore, right?